### PR TITLE
Removed field thngs from Collection.

### PR DESCRIPTION
**Pull request by sollecitom**
Engine 1069

### CODE REVIEW

File: thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Collection.java [hunk 1]
```diff
@@ -16,10 +14,6 @@
 	private static final long serialVersionUID = -2064399431964890923L;
 	private String name;
 	private String description;
-	/**
-	 * Collection of {@link Thng#id} references.
-	 */
-	private LinkedHashSet<String> thngs;
```
Comment (coffeemachine) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Collection.java]: Breaking API change detected.

File: thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java [hunk 2]
```diff
@@ -110,4 +117,14 @@ public String firstIdentifier() {
 
 		return identifiers.values().iterator().next();
 	}
+
+	public Set<String> getCollections() {
+
+		return collections != null ? new TreeSet<>(collections) : null;
```
Comment (coffeemachine) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: Avoid logic in this class.
This is inconsistent with other collection fields. You can't do an update with this field set to null.
Also, why a tree set?

File: thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java [hunk 3]
```diff
@@ -110,4 +117,14 @@ public String firstIdentifier() {
...
+	}
+
+	public void setCollections(final Set<String> collections) {
+
+		this.collections = collections != null ? new TreeSet<>(collections) : null;
```
Comment (coffeemachine) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: Avoid logic in this class.

Comment (sollecitom) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: Fixed

File: thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java (hunk 2 above)
Comment (sollecitom) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Thng.java]: Fixed

File: thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Collection.java (hunk 1 above)
Comment (sollecitom) [inline thng-resource-model/src/main/java/com/evrythng/thng/resource/model/store/Collection.java]: As discussed, it is.

**Pull request merged**